plothist/hep_plotters.py

```diff
@@ -165,20 +195,28 @@ def compare_data_mc(
     ax_ratio.bar(
         x=mc_hist_total.axes[0].centers,
         bottom=np.nan_to_num(1 - scaled_mc_uncertainty, nan=0),
-        height=np.nan_to_num(2 * scaled_mc_uncertainty, nan=100),
+        height=(
+            np.nan_to_num(2 * scaled_mc_uncertainty, nan=100)
+            if ratio == "diff"
+            else np.nan_to_num(2 * scaled_mc_uncertainty, nan=0)
+        ),
         width=mc_hist_total.axes[0].widths,
         edgecolor="dimgrey",
         hatch="////",
         fill=False,
         lw=0,
     )
 
-    ax_ratio.axhline(1, ls="--", lw=1.0, color="black")
-    ax_ratio.set_ylim(0.0, 2.0)
+    if ratio == "diff":
+        ax_ratio.axhline(1, ls="--", lw=1.0, color="black")
+        ax_ratio.set_ylabel(r"$\frac{Data}{Pred.}$")
+        ax_ratio.set_ylim(0.0, 2.0)
+    elif ratio == "pull":
+        ax_ratio.axhline(0, ls="--", lw=1.0, color="black")
+        ax_ratio.set_ylim(-5.0, 5.0)
```

Comment:
We should let the user define the range, this was asked by a German user. If `None` is provided, we set it to (0,2) or (-5,5) depending on `compare`.